package.yaml

```diff
@@ -65,6 +65,7 @@ ghc-options:
   - -Wno-all-missed-specialisations
   - -Wno-prepositive-qualified-module
   - -Wno-monomorphism-restriction
+  - -optP-Wno-nonportable-include-path
```

Comment:
In macOS, the project builds successfully but it shows an error message. It seems to correspond to a currently open issue for cabal, and adding this GHC option is a workaround for avoiding the message:

https://github.com/haskell/cabal/issues/4739

Is it ok if I include it in this PR?

Reply:
Okay, sounds fine.

Please leave a brief comment mentioning what this option achieves, and I would also like to see it in a separate commit as it is unrelated to the main logic.

Reply:
Done! Is this comment style appropriate for a stack file?

Reply:
Looks good :+1: 

A minor suggestion: it may be beneficial to also include the link where that solution was suggested, but it's really optional.

Reply:
Very good, thank you.